tests/test_onnx/test_head.py

```diff
@@ -224,3 +216,292 @@ def test_yolov3_head_get_bboxes():
     for i in range(len(onnx_outputs)):
         np.testing.assert_allclose(
             torch_outputs[i], onnx_outputs[i], rtol=1e-03, atol=1e-05)
+
+
+def fcos_config():
+    """FCOS Head Config."""
+    test_cfg = mmcv.Config(
+        dict(
+            deploy_nms_pre=0,
+            min_bbox_size=0,
+            score_thr=0.05,
+            nms=dict(type='nms', iou_threshold=0.5),
+            max_per_img=100))
+
+    model = FCOSHead(num_classes=4, in_channels=1, test_cfg=test_cfg)
+
+    model.requires_grad_(False)
```

Comment:
Why set to `False` for model?

Reply:
set `False` for `requires_grad` is to freeze the model parameters

Reply:
But you already use `with torch.no_grad()` in `ort_validate`.  Please verify if it is necessary to explicitly set `requires_grad` to `False`.